src/pages/help/faq/index.mdx

```diff
@@ -5,16 +5,25 @@ description:
   Here, we answer some of our frequently asked questions.
 ---
 
-<PageDescription>
+<AnchorLinks>
 
-What is the Carbon Design System?
+<AnchorLink>About Carbon</AnchorLink>
+<AnchorLink>Contributing</AnchorLink>
+<AnchorLink>Creating with Carbon</AnchorLink>
 
-</PageDescription>
+</AnchorLinks>
+
+## All about Carbon
+
+### What is the Carbon Design System?
 
 Carbon is the open-source design system for all IBM software products. It is a
 series of individual styles, components, and guidelines used for creating
 unified product experiences.
 
+To learn more about how we work check out
+[All about Carbon](/all-about-carbon/what-is-carbon).
+
 ### Who works on Carbon?
```

Comment:
Let's take out the sentence about the Austin studio. (I can't edit that paragraph for some reason.)

Reply:
Done!